### ISSUE TITLE
non-whitespace and non-commented content is forbidden outside server-block

### ISSUE THREAD

**Issue by aenglert42** [TamLem/Webserv#18]
The following config file triggers above mentioned error:
```
server {
	listen 8080
	root /ourTester/ourTesterRoot/
	server_name localhost:8080

# 	location / {
# 		method GET
# 	}

# 	location /put_test/ {
# 		method PUT; save files to a directory of your choice
# 	}

# 	location *.bla {
# 		method POST; by calling the cgi_test executable
# 	}

# # /post_body must answer anything to POST request with a maxBody of 100

# 	client_max_body_size 100; MB?
# 	client_body_buffer_size 100; MB?

# 	location /directory/ {
# 		method GET
# 		root /42tester/YoupiBanane/
# 		index_page youpi.bad_extension
# 	}
}
```

**Comment by aenglert42** [comment]
Seems like it happens if the file doesn't end with an empty line.

**Comment by tblaase** [comment]
fixed with 8b818b98078eea42f5c4bc6157a89012fb86348d
